Formula/toml-test.rb

```diff
@@ -0,0 +1,21 @@
+class TomlTest < Formula
+  desc "Language agnostic test suite for TOML parsers"
+  homepage "https://github.com/burntsushi/toml-test"
+  url "https://github.com/BurntSushi/toml-test/archive/refs/tags/v1.2.0.tar.gz"
+  sha256 "33bf4e9c017cd57f3602e72d17f75fb5a7bcc7942541c84f1d98b74c12499846"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+
+    (share/"toml-test").install "tests"
+  end
+
+  test do
+    system "#{bin}/toml-test", "-version"
+    system "#{bin}/toml-test", "-help"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Done.